restaking_core/src/ncn.rs

```diff
@@ -30,11 +30,11 @@ pub struct Ncn {
     /// The slasher admin of the NCN
     pub slasher_admin: Pubkey,
 
-    /// The withdraw admin of the NCN
-    pub withdraw_admin: Pubkey,
+    /// The delegate admin of the NCN
+    pub delegate_admin: Pubkey,
 
-    /// The withdraw fee wallet of the NCN
-    pub withdraw_fee_wallet: Pubkey,
+    // Reserved space
```

Comment:
Note for @buffalu - I think we should clean up the reserved_1 before official launch